staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -91,8 +91,8 @@ func (m *CRConverterFactory) NewConverter(crd *apiextensionsv1.CustomResourceDef
 	return &safeConverterWrapper{unsafe}, unsafe, nil
 }
 
-// crConverterInterface is the interface all cr converters must implement
-type crConverterInterface interface {
+// Converter is the interface all cr converters must implement
+type Converter interface {
```

Comment:
nit: unsure about the name. the fact that this is converting custom resources is mentioned only once in "all cr converters". Can we call this `CRDConverters`?

Reply:
I was thinking that given it's in the apiextension-apiserver module, Converter would be sufficient, but if you think it might be ambiguous, we can call it CRConverter